Dockerfile-slim.template

```diff
@@ -19,11 +38,12 @@ RUN set -ex \
     && deps=" \
        gdal-bin \
        gettext \
-       postgresql-client=${PG_VERSION} \
+       postgresql-client-$PG_MAJOR=$PG_VERSION \
     " \
-    && apt-get update && apt-get install -y ${buildDeps} ${deps} --no-install-recommends \
+    && apt-get update && apt-get install -y $buildDeps $deps --no-install-recommends \
     && pip install --no-cache-dir -r /tmp/requirements.txt \
-    && apt-get purge -y --auto-remove ${buildDeps} \
+    && apt-get purge -y --auto-remove $buildDeps \
+       $(! command -v gpg > /dev/null || echo 'gnupg dirmngr') \
```

Comment:
What is the purpose of this?

Reply:
Removes `gnupg` and `dirmngr` if `gpg` exists (exits non-zero).